docs/sources/setup/install/helm/install-monolithic/_index.md

```diff
@@ -10,107 +10,411 @@ weight: 100
 
 # Install the monolithic Helm chart
 
-This Helm Chart installation runs the Grafana Loki *single binary* within a Kubernetes cluster.
+This Helm Chart installation deploys Grafana Loki in [monolithic mode](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/deployment-modes/#monolithic-mode) within a Kubernetes cluster.
 
-If you set the `singleBinary.replicas` value to 1 and set the deployment mode to `SingleBinary`, this chart configures Loki to run the `all` target in a [monolithic mode](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/deployment-modes/#monolithic-mode), designed to work with the filesystem storage configuration. It will also configure meta-monitoring of metrics and logs. 
+## Prerequisites
+
+- Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
+- A running Kubernetes cluster.
+
+## Single Replica or Multiple Replicas
+
+There are two ways to deploy Loki in monolithic mode:
+1. **Single Replica**: Run Loki with a single replica. This mode is useful for testing and development or if you are planning to run Loki as a meta-monitoring system.
+2. **Multiple Replicas**: Run Loki with multiple replicas. This mode is useful for high availability and scalability. This mode is less economical than distributed mode, but it is simpler to operate.
+
+Once you have selected choose the appropriate `values.yaml` configuration file below and then continue with the deployment steps.
```

Comment:
Once you have selected..... What?  
(Once you have selected how you want to deploy Loki....)